src/GeositeFramework/sample_plugins/identify_point/main.js

```diff
@@ -60,6 +70,13 @@ define(["dojo/_base/declare", "framework/PluginBase"],
                             break;
                     }
                 });
+
+                // Call `renderSampleImageModal` method on clicking thumbnail
+                $('.identify-point-plugin-thumbnail-image').on('click', function() {
+                    $thumbnailImage = $('#sample-plugin-thumbnail');
```

Comment:
Also going to replace the `this.src` assignment with

```js
var modalImageSource = 'sample_plugins/identify_point/FutureHabitat_c.jpg';
```

because it's slightly less opaque.